docs/api/environments/lbf.md

```diff
@@ -0,0 +1,8 @@
+::: jumanji.environments.routing.robot_warehouse.env.LevelBasedForaging
```

Comment:
This should fix the failing CI :wink: 

```suggestion
::: jumanji.environments.routing.lbf.env.LevelBasedForaging
```

Reply:
I remember fixing that but I don't know where I pushed it thanks 🫠

Reply:
It doesn't fix the issue 😢

Reply:
Sorry I fixed it in another PR, should work now :smile: